README.md

```diff
@@ -52,11 +52,11 @@ a server, an admin tool and a client Web UI to see the captured lineage.
 There are two ways how to do it:
 
 #### Download prebuild Spline artifacts from the Maven repo
--   [```za.co.absa.spline:admin:0.4.0```](https://repo1.maven.org/maven2/za/co/absa/spline/admin/0.4.0/)
+-   [```za.co.absa.spline:admin:0.5.0```](https://repo1.maven.org/maven2/za/co/absa/spline/admin/0.5.0/)
```

Comment:
why is this surrounded with 3 ``` ` ```. Wouldn't one suffice?

Reply:
Seems so, it just was there from the beginning :) I only replaced 0.4 with 0.5 in this file